### PR TITLE
Encode params for cleanse API the same way we do for other APIs

### PR DESCRIPTION
**Pull request by naikipl**
Code change for issue #10

### CODE REVIEW

File: spec/lib/addressfinder/cleanse_spec.rb [hunk 1]
```diff
@@ -98,7 +98,7 @@
     context 'with a question mark value' do
       let(:q){ '?' }
 
-      it { expect(encoded_params).to eq('q=%3F&format=json&key=XXX&secret=YYY') }
+      it { expect(encoded_params).to eq('q=?&format=json&key=XXX&secret=YYY') }
```
Comment (nigelramsay) [inline spec/lib/addressfinder/cleanse_spec.rb]: that question mark appears incorrect. I would have thought it should have been encoded as the %3F

Comment (naikipl) [inline spec/lib/addressfinder/cleanse_spec.rb]: All the other API modules are happy with `URI.encode`. The alternative is to change all the others to use `ERB::Utils.url_encode` and then add an erb dependency. Why is it important that ? is encoded. I would have thought URI.encode would do as much as we needed.

Comment (naikipl) [inline spec/lib/addressfinder/cleanse_spec.rb]: Maybe it is `CGI::escape` we want. Is `URI.encode` deprecated?

Comment (naikipl) [inline spec/lib/addressfinder/cleanse_spec.rb]:
```
query_params = params.map do |k,v|
  "#{k}=#{CGI::escape(v)}"
end		
query_params.join('&')
```

maybe?

Comment (nigelramsay) [inline spec/lib/addressfinder/cleanse_spec.rb]: Give it a go. The main thing is that the question mark above gets encoded

**Comment by nigelramsay** [comment]
Looks good @naikipl 

Can you ship a new version? You'll need to update the version number and tweak the ChangeLog file

**Pull request merged**